prow/plugins/plugins.go

```diff
@@ -138,6 +138,43 @@ type Configuration struct {
 	Slack           Slack               `json:"slack,omitempty"`
 	// ConfigUpdater holds config for the config-updater plugin.
 	ConfigUpdater ConfigUpdater `json:"config_updater,omitempty"`
+	Blockades     []Blockade    `json:"blockades,omitempty"`
+}
+
+/*
+  Blockade specifies a configuration for a single blockade.blockade. The configuration for the
+  blockade plugin is defined as a list of these structures. Here is an example of a complete
+  yaml config for the blockade plugin that is composed of 2 Blockade structs:
+
+	blockades:
+	- repos:
+	  - kubernetes-incubator
+	  - kubernetes/kubernetes
+	  - kubernetes/test-infra
+	  blockregexps:
+	  - "docs/.*"
```

Comment:
use ' to quote regexps to avoid interpreting \ escapes

Reply:
Is that valid yaml? Back ticks don't appear to be a valid replacement for quotes.

Reply:
not back ticks, single quotes

Reply:
Ah ok. Fixed.